common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -2214,10 +2216,19 @@ object CelebornConf extends Logging {
       .bytesConf(ByteUnit.BYTE)
       .createWithDefaultString("64mb")
 
+  val ESTIMATED_PARTITION_SIZE_MAX_SIZE: OptionalConfigEntry[Long] =
+    buildConf("celeborn.master.estimatedPartitionSize.maxSize")
+      .categories("master")
+      .doc(
+        "Max partition size for estimation.")
```

Comment:
Adds the description of the default value, which is partitionSplitMaximumSize * 2.